test/dom-parsing/__snapshots__/xmlConformance.tests.ts.snap

```diff
@@ -929,11 +929,11 @@ At line 1, character 16:
 `;
 
 exports[`XML Conformance Test Suite IBM XML 1.0 Tests not-wf ibm-not-wf-P24-ibm24n02.xml - Tests VersionInfo with a required field missing. The white space is missing between the key word "xml" and the VersionInfo in the XMLDecl.: ibm-not-wf-P24-ibm24n02.xml 1`] = `
-"Parsing document failed, expected \\"whitespace\\"
-At line 1, character 6:
+"Parsing document failed, expected \\"?>\\"
```

Comment:
A prettier error would be that the PI is unparseable because there's a `=` in the target. But this works.

Reply:
It ends up with this error because the target Name can not contain the = so it ends just before that character. At some point I'd like to look into having prsc report on optionals that were rejected at the point of the error (which would restore the reporting of the "whitespace" alternative), but for now I think this is fine.